Tasks/BashV3/_buildConfigs/Node20/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "vsts-tasks-bash",
+  "version": "1.0.0",
+  "description": "Azure Pipelines Bash Task",
+  "main": "bash.js",
+  "scripts": {
+    "build": "node ../../make.js build --task BashV3",
+    "test": "node ../../make.js test --task BashV3"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/Microsoft/azure-pipelines-tasks.git"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/azure-pipelines-tasks/issues"
+  },
+  "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
+  "dependencies": {
+    "@types/mocha": "^9.1.1",
+    "@types/node": "^20.3.1",
+    "azure-pipelines-task-lib": "^5.0.0-preview.0",
+    "azure-pipelines-tasks-utility-common": "^3.225.1",
```

Comment:
todo: azure-pipelines-tasks-utility-common is node16, needs to be updated.  not for this pr